packages/preview-middleware-client/src/flp/bootstrap.js

```diff
@@ -9,14 +9,18 @@
  * @param fnCallback {Function} The callback function to be executed after the bootstrap is loaded.
  */
 async function ushellBootstrap(fnCallback) {
-    let src = '/test-resources/sap/ushell/bootstrap/sandbox.js';
+    const currentPath = location.pathname.substring(0, location.pathname.lastIndexOf('/'));
```

Comment:
Not sure if that's the best way to do this but at least it seems to work with cds-plugin-ui5 as well as without it (unit tests are still green 🐱). Basic idea taken from [testsuite.qunit.js template](https://github.com/SAP/open-ux-tools/blob/main/packages/preview-middleware/templates/test/testsuite.qunit.js#L3)

Reply:
Not beautiful but I don't have a smart suggestion, so I am ok with it.